measuring-client/src/main/java/de/cyface/app/ui/button/DataCapturingButton.java

```diff
@@ -901,50 +912,50 @@ public void onFixLost() {
     @Override
     public void onNewGeoLocationAcquired(GeoLocation geoLocation) {
```

Comment:
in our app the duration of an ongoing measurement is only updated on new location (in this test app here) but that's enough for testing purposes, the SR App can pull the `duration` also each second as they do right now I guess.